src/azure-cli/azure/cli/command_modules/sql/commands.py

```diff
@@ -345,6 +347,26 @@ def load_command_table(self, _):
         g.command('delete', 'delete')
         g.custom_command('set-primary', 'failover_group_failover')
 
+    ###############################################
+    #             sql instance-pool               #
+    ###############################################
+
+    instance_pools_operations = CliCommandType(
+        operations_tmpl='azure.mgmt.sql.operations#InstancePoolsOperations.{}',
+        client_factory=get_sql_instance_pools_operations)
+    with self.command_group('sql instance-pool', instance_pools_operations, client_factory=get_sql_instance_pools_operations, is_preview=True) as g:
+        g.command('show', 'get',
+                  table_transformer=instance_pool_table_format)
+        g.custom_command('list', 'instance_pool_list',
+                         table_transformer=instance_pool_table_format)
+        g.command('update', 'update')
+        g.command('delete', 'delete', supports_no_wait=True, confirmation=True)
+        g.custom_command(
```

Comment:
should create also have table_transformer?

Reply:
Doesn't really help all that much. It returns only one object and all the details are clearly visible.